### PR TITLE
fix: `COLOR_INCLUDED` to colorize `included` output

### LINKED ISSUE #83663: `COLOR_SKIP` also alters `included` tasks color display
### Summary

I noticed that both `included:` and `skipped:` tasks in the `stdout` of the `ansible-playbook` command share the same color setting: `COLOR_SKIP`

We have a "master" playbook to configure our Ubuntu hosts - with lots of `include_role` / `include_tasks` modules tied to `when` conditions. I wanted to have a different COLOR depending of the module being executed or skipped, to detect more easily what is being included and what is being skipped. To my surprise, `COLOR_SKIP` alters both outcomes, making it a bit hard to differentiate via colors.

Can confirm by looking at the code base `COLOR_SKIP` is intentionally set for `included`
https://github.com/ansible/ansible/blob/79f819dc54e4cf5dc3853aea6b39626ee5e6fef4/lib/ansible/plugins/callback/default.py#L291-L296

Note sure what's the rationale behind it but I don't see why we cannot set a specific color for `included` tasks. I would happily implement this on my spare time.

### Issue Type

Bug Report

### Component Name

callback

### Ansible Version

```console
$ ansible --version

2.15.5 - but assuming it's impacting pretty much all ansible version
```

### Configuration

```console
# if using a version older than ansible-core 2.12 you should omit the '-t all'
$ ansible-config dump --only-changed -t all | grep COLOR
ANSIBLE_FORCE_COLOR(/Users/toto/ansible/ansible.cfg) = True
COLOR_CHANGED(/Users/toto/ansible/ansible.cfg) = yellow
COLOR_CONSOLE_PROMPT(/Users/toto/ansible/ansible.cfg) = white
COLOR_DEBUG(/Users/toto/ansible/ansible.cfg) = green
COLOR_DEPRECATE(/Users/toto/ansible/ansible.cfg) = purple
COLOR_DIFF_ADD(/Users/toto/ansible/ansible.cfg) = green
COLOR_DIFF_LINES(/Users/toto/ansible/ansible.cfg) = cyan
COLOR_DIFF_REMOVE(/Users/toto/ansible/ansible.cfg) = red
COLOR_ERROR(/Users/toto/ansible/ansible.cfg) = red
COLOR_HIGHLIGHT(/Users/toto/ansible/ansible.cfg) = white
COLOR_OK(/Users/toto/ansible/ansible.cfg) = green
COLOR_SKIP(/Users/toto/ansible/ansible.cfg) = cyan
COLOR_UNREACHABLE(/Users/toto/ansible/ansible.cfg) = bright red
COLOR_VERBOSE(/Users/toto/ansible/ansible.cfg) = blue
COLOR_WARN(/Users/toto/ansible/ansible.cfg) = bright purple
```

### OS / Environment

Mac OS Ventura 13.6.2 - but also seen on Ubuntu 22. 
I believe it's not related to OS or envrionment.

### Steps to Reproduce

`COLOR_SKIP=magenta` vs `COLOR_SKIP=cyan`

image

### Expected Results

I would expect a new config field to set "included" tasks color output , e.g. `COLOR_INCLUDED` - so we can clearly differentiate (with colors) in the output what is being skipped - what is being included

image

### Actual Results

```console
See steps to reproduce
```

### Code of Conduct

- [X] I agree to follow the Ansible Code of Conduct

### PR DESCRIPTION
**Pull request by akire0ne**
##### SUMMARY

Fixes: https://github.com/ansible/ansible/issues/83663

##### ISSUE TYPE

- Bugfix Pull Request

##### ADDITIONAL INFORMATION

Adds a new parameter to set color for Included tasks.
 Prior to that, the included tasks used the same color as `COLOR_SKIP` parameter. This made differentiating the `included` and `skipped` tasks a bit hard in some occurrence.
`COLOR_INCLUDED` will be defaulted to previous hardcoded value `cyan` - so this change shouldnt have an impact on current deployments.

### CODE REVIEW

**Comment by akire0ne** [comment]
Not all colors seems to work with this... I will investigate a bit before putting this "Ready for review" ...

with `ANSIBLE_COLOR_INCLUDED=red` , everything OK

image

with `ANSIBLE_COLOR_INCLUDED=magenta`, it works but displays a Warning ?

image

**Comment by bcoca** [comment]
it is not about the color, but the config and it's value. this happens in the 'color to log level' mapping, which does not contain the new color config, https://github.com/ansible/ansible/blob/devel/lib/ansible/utils/display.py#L172

**Comment by akire0ne** [comment]
Indeed this seems to do the trick @bcoca 
image

**Comment by bcoca** [comment]
related #83712, this should 'fix' the issue also, but keeping color JIC people are using display in 'fun' ways, will incorporate new config from this ticket once merged.

File: lib/ansible/config/base.yml [hunk 1]
```diff
@@ -304,6 +304,13 @@ COLOR_HIGHLIGHT:
   env: [{name: ANSIBLE_COLOR_HIGHLIGHT}]
   ini:
   - {key: highlight, section: colors}
+COLOR_INCLUDED:
+  name: Color for 'included' task status
+  default: cyan
+  description: Defines the color to use when showing 'Included' task status.
+  env: [{name: ANSIBLE_COLOR_INCLUDED}]
+  ini:
+  - {key: included, section: colors}
```
Comment (bcoca) [inline lib/ansible/config/base.yml:313]: needs version_added

**Review by bcoca — CHANGES_REQUESTED** [review]
Also this requires a changelog

File: lib/ansible/config/base.yml (hunk 1 above)
Comment (akire0ne) [reply lib/ansible/config/base.yml:313]: All done

**Review by bcoca — APPROVED** [review]

**Pull request merged**